### DISCUSSION TITLE
20221107(월) 백엔드 주간회의

### DISCUSSION THREAD

**Discussion by hunjin-choi** [innovationacademy-kr/42dashboard#509] [Backend Progress]
1. 이번주 계획

* dcho: 이번주내로 스태프 권한 이슈 점검 및 보완. 로그인 코드에서 보안이슈가 발생할 만한 부분 점검 및 보완.
* junghan: 42서울 스프레드시트에 준하는 목업스프레드 시트 만들기. any타입으로 된 코드들 정확한 데이터타입으로 변경(리팩터링)
* huchoi: 테이블 역정규화 고민. 테이블 조인 문제 고민. 
* doyun: any타입으로 된 코드들 정확한 데이터타입으로 변경(리팩터링). 코드양이 상당히 많아서 복습, 분석 하고 수정할 만한 부분 수정.

**Comment by DBrider3** [comment]
화이팅!
